packages/api-headless-cms/src/plugins/models/environmentAlias.model.ts

```diff
@@ -8,16 +8,28 @@ import {
     ref,
     withName,
     withHooks,
-    withProps
+    withProps,
+    onSet
 } from "@webiny/commodo";
+import slugify from "slugify";
+
+const toSlug = text =>
```

Comment:
Just figured we are using the same thing in `packages/api-headless-cms/src/content/plugins/models/contentModelGroup.model.ts:15`. Please extract this function into a separate file, so we can reuse the same thing in both cases.

You can put it here:
![image](https://user-images.githubusercontent.com/5121148/85128116-f661cb80-b230-11ea-9fc9-74cd3699dc0f.png)